### DISCUSSION TITLE
Recurring Tasks in Task Planning extension

### DISCUSSION THREAD

**Discussion by mbiolsi** [kimai/Paid-plugins#139] [Questions & Answers]
Is there a way to enable recurring tasks in the Task Planning extension? For instance, I have a monthly task for a client - can I enter it and then make it recurring for the year or do I have to enter each task individually?

**Comment by pbksol** [comment]
There's no support for recurring tasks. What you can do is create all tasks for a year, a month or so via API automatically. Or another approach is to check if a task is done and then create a new one. To filter recurring from non recurring ones using tags for example could be a way to go.
